### DISCUSSION TITLE
retrieving failed logs from travis runs

### DISCUSSION THREAD

**Discussion by duncdrum** [eXist-db/exist#4871] [Team Posts]
Travis now dumps the test logs of failed runs to [chunk.io](http://chunk.io).
```yaml
after_failure:
   - tar -cf logs.zip test/junit/data
   - curl -u $CHUNK_USER:$CHUNK_PW -sT logs.zip chunk.io
```

Merged PRs, and builds on branches are enabled automatically. You'll find the URL for to retrieve the logs on [travis](https://travis-ci.com/duncdrum/exist/jobs/163536359) at the very end of the log of each failed testrun:

For security reasons, encrypted env variables are disabled for PR builds by travis. 
To activate the same feature in your fork, you should make sure that the variable `$CHUNK_USER` and `$CHUNK_PW` are set on your accounts.  This entails getting an account at [chunk.io](http://chunk.io)

You can add them at: `https://travis-ci.com/$$YOUR_GH_USER_NAME$$/exist/settings`

**Comment by adamretter** [comment]
Nice, thanks Duncan.

**Comment by dizzzz** [comment]
nice !!!!!!
